.gitignore

```diff
@@ -1,8 +1,6 @@
 tmp
 node_modules
-src/config.json
-.DS_Store
```

Comment:
I don't think it's necessary for you to remove these from here given they are already there. Not everyone has a global git ignore setup so personally I prefer this approach.